apps/ledger-live-desktop/src/renderer/screens/stake/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect, useMemo } from "react";
+import SelectAccountAndCurrencyDrawer from "~/renderer/drawers/DataSelector/SelectAccountAndCurrencyDrawer";
+import { setDrawer } from "~/renderer/drawers/Provider";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { Account } from "@ledgerhq/types-live";
+import perFamilyManageActions from "~/renderer/generated/AccountHeaderManageActions";
+import { useHistory } from "react-router-dom";
+import { stakeDefaultTrack } from "./constants";
+import { track, page } from "~/renderer/analytics/segment";
+
+type Props = {
+  account?: Account;
+  parentAccount?: Account;
+};
+
+const StakeFlow = (props: Props) => {
```

Comment:
the react convention is to start hook names with `use`... so this should be called something like `useStakeFlow`;